setup.cfg

```diff
@@ -28,6 +28,7 @@ test =
     pytest-cov
     cython
     coverage
+    setuptools;python_version>='3.12'
```

Comment:
@bsipocz , I needed this for the Python 3.12-dev job to pass. Does this look right or is this a symptom of something else?